src/Node/Stream/StdIO.purs

```diff
@@ -0,0 +1,11 @@
+-- | The standard IO streams for the current process.
+module Node.Stream.StdIO where
```

Comment:
I think that makes sense. Once the API is nailed down, things like gzip and crypto streams would probably make nice libraries too.